### PR TITLE
Remove a duplicate column from information_schema

### PR DESCRIPTION
**Pull request by pgavlin**
Just what it says on the tin. This duplicate column causes problems for DuckDB when attempting to connect to doltdb databases.

### CODE REVIEW

**Comment by timsehn** [comment]
Thanks for the submission. @fulghum will review further

**Comment by fulghum** [comment]
Hi @pgavlin, thank you for identifying this issue and sending us a contribution. 🙏  btw... I love the Pulumi hat in your profile pic! 

I noticed that MySQL's schema for `view_routine_usage` includes a `SPECIFIC_SCHEMA` column, instead of the duplicate `SPECIFIC_CATALOG` column, so I went ahead and created https://github.com/dolthub/go-mysql-server/pull/2584 with that change, just to get the fix out faster for you, instead of asking you to update your PR. I'll wait an hour or so before I merge in that other PR, just in case you do want to update the PR so you can get the commit honors. 😉 

Major thanks for helping us find this one! 🙏 

Also, sounds very cool that you're using DuckDB together with Dolt. Please let us know if you hit any more snags or MySQL incompatibilities and we'll be happy to help out. Don't hesitate to let us know anything we can do to help you build with Dolt!

File: sql/information_schema/information_schema.go [hunk 1]
```diff
@@ -771,7 +771,6 @@ var viewRoutineUsageSchema = Schema{
 	{Name: "TABLE_SCHEMA", Type: types.MustCreateString(sqltypes.VarChar, 64, Collation_Information_Schema_Default), Default: nil, Nullable: true, Source: ViewRoutineUsageTableName},
 	{Name: "TABLE_NAME", Type: types.MustCreateString(sqltypes.VarChar, 64, Collation_Information_Schema_Default), Default: nil, Nullable: true, Source: ViewRoutineUsageTableName},
 	{Name: "SPECIFIC_CATALOG", Type: types.MustCreateString(sqltypes.VarChar, 64, Collation_Information_Schema_Default), Default: nil, Nullable: true, Source: ViewRoutineUsageTableName},
```
Comment (fulghum) [inline sql/information_schema/information_schema.go:773]:
```suggestion
	{Name: "SPECIFIC_CATALOG", Type: types.MustCreateString(sqltypes.VarChar, 64, Collation_Information_Schema_Default), Default: nil, Nullable: true, Source: ViewRoutineUsageTableName},
	{Name: "SPECIFIC_SCHEMA", Type: types.MustCreateString(sqltypes.VarChar, 64, Collation_Information_Schema_Default), Default: nil, Nullable: true, Source: ViewRoutineUsageTableName},
```

**Review by fulghum — APPROVED** [review]
Looks good! 🚢

**Comment by fulghum** [comment]
Actually... I forgot I could edit the remote PR branch on GitHub, so I went ahead and applied the change here so that we could merge in your PR. 😉 

Thanks again for reporting this problem and submitting a contribution. Sorry it took us a couple of days to get to it – we typically try to turn around contributions faster than that.

**Comment by pgavlin** [comment]
> btw... I love the Pulumi hat in your profile pic!

hah, thanks! good reminder to update my GH profile to make it clearer that I work there :)

**Pull request merged**